src/main/java/jalau/at18/azul/view/FactoryLabel.java

```diff
@@ -2,17 +2,19 @@
 
 
 import java.awt.*;
-
 import javax.swing.JLabel;
 
 public class FactoryLabel extends JLabel {
-    private static final int OVAL_WIDTH = 90;
-    private static final int OVAL_HEIGHT = 130;
+    private static final int OVAL_WIDTH = 100;
+    private static final int OVAL_HEIGHT = 150;
+    public static final int NUMBER_OF_FACTORIES = 5;
```

Comment:
Is `NUMBER_OF_FACTORIES` being used? If this is not the case, it should be removed.

Reply:
Already remove the number_of_factories